src/System.Net.NameResolution/src/System/Net/DNS.cs

```diff
@@ -112,8 +112,11 @@ internal static IPHostEntry InternalGetHostByAddress(IPAddress address, bool inc
 
                     if (NetEventSource.Log.IsEnabled())
                     {
-                        NetEventSource.Exception(NetEventSource.ComponentType.Socket, "DNS",
-                        "InternalGetHostByAddress", new InternalSocketException(errorCode, nativeErrorCode));
+                        NetEventSource.Exception(
+                            NetEventSource.ComponentType.Socket, 
+                            "DNS",
+                            "InternalGetHostByAddress", 
```

Comment:
Nit: As long as you're changing things here, you could use nameof to avoid the hardcoded strings.